tests/unit/test_hydration_manager_parallel.py

```
@@ -0,0 +1,59 @@
+
+class _SlowProvider(StateProvider[Any]):
+    def __init__(self, delay: float = 1.0) -> None:
+        self.delay = delay
+        self.load_calls: list[str] = []
+
+    async def load(self, key: str) -> Any:
+        self.load_calls.append(key)
+        await asyncio.sleep(self.delay)
+        return {"key": key}
+
+    async def save(self, key: str, data: Any) -> None:
```

Ruff (0.14.7)

Unused method argument: `key`

(ARG002)